helper/cli.rs

```diff
@@ -51,6 +51,7 @@ pub async fn parse_cli() -> Option<Settings> {
 
     if !credentials.is_complete()
     {
+        eprintln!("===> Incomplete credits!");
```

Comment:
debug prints in this file are not needed i guess :)

Reply:
Well, other paths did have debug prints, and those were useful so I think it is a good idea to add them.

Reply:
Ah yes you added those in another PR. Whatever :) Then please at least reword this to `credentials`.

Reply:
Fixed.